tests/integration-service/integration.go

```diff
@@ -151,16 +148,18 @@ var _ = framework.IntegrationServiceSuiteDescribe("Integration Service E2E tests
 
 				createApp()
 				createComponent()
-
 				_, err = f.AsKubeAdmin.IntegrationController.CreateIntegrationTestScenario(applicationName, appStudioE2EApplicationsNamespace, BundleURL, InPipelineName)
+				// create a integrationTestScenario v1beta1 version works also here
+				// ex: _, err = f.AsKubeAdmin.IntegrationController.CreateIntegrationTestScenario_beta1(applicationName, appStudioE2EApplicationsNamespace, gitURL, revision, pathInRepo)
```

Comment:
if it works with the new version as the comment says, should v1beta1 replace the v1alpha1? is there a reason to keep using v1alpha1?

Reply:
It is just to make sure it's still compatible to v1alpha1 version .